Week-2/3.Weather-wear.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+function clothingGuru(x) {
+  if (!x) {
+    return "Invalid Input";
+  }
+
+  if (x > 0 && x <= 20) {
+    return "Sweater and Jeans";
```

Comment:
nice! ok na 'to

my style is one return sa last if not error, pero this is good na